src/Components/Main/Pages/MessagesPage/MessageForm/MessageFormContainer.js

```diff
@@ -0,0 +1,29 @@
+import { Formik } from 'formik';
+import * as Yup from 'yup';
+import { MessageForm } from './MessageForm';
+
+const validationSchema = Yup.object().shape({
+  text: Yup.string()
+    .max(100, 'Must be shorter than 100 characters')
```

Comment:
"Must be shorter than 100 characters" means 0-99, but 100 is allowed as well, so message may be wrong here =)
Maybe change to "Must be no more than 100 characters" =)

Reply:
Done!